### ISSUE TITLE
Docs not updated to show change to config file

### ISSUE THREAD

**Issue by immesys** [ogen-go/ogen#1282]
### What version of ogen are you using?

1.2.2

### Can this issue be reproduced with the latest version?

Yes

### What did you do?

The docs say I can use --convenient-errors=on (which I was using in older versions) but this doesn't work against the latest. Doing a git bisect it seems like there is now a config file, but the docs don't mention that.

**Comment by volkerhess2** [comment]
You can create a yaml file with 
```
generator:
 convenient_errors: false
```
and use it with the config flag: --config=myconfig.yaml

**Comment by abemedia** [comment]
I agree it would be great to have a page dedicated to the config file on the docs. I ended up looking at the ogen code to figure out the possible config values: https://github.com/ogen-go/ogen/blob/4182357878201caf52384c2acd887b44d02a3a1f/gen/options.go#L153-L175
